HTShowcase/HTShowcase/Coordinators/ShowcaseCoordinator.swift

```diff
@@ -25,6 +25,12 @@ class ShowcaseCoordinator: BaseCoordinator<Void> {
         window.rootViewController = navigationController
         window.makeKeyAndVisible()
         
+        _ = viewModel.didSelectApp.do(onNext: {(app) -> Void in
```

Comment:
instead of declaring this as _ you should directly subscribe to it and add disposed(by) i think